source/install.txt

```diff
@@ -101,6 +101,11 @@ Procedure
       
             .. include:: /includes/steps/install-shell-linux-generic.rst
 
+.. important:: Windows Users
```

Comment:
I think we can move this so it only appears under the Windows tab.

Reply:
Do we need a parallel blurb for Linux / macos tabs which explain where to find Preferences on those operating systems as well?

Reply:
Moved it under the windows tab and changed it to a note. Judging from the Jira ticket, this is a behavioral oddity which is only noteworthy for windows users.